app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,146 @@
+<div>
+  <div class="left-float">
+    <h2> Event Name  </h2>
+  </div>
+  <div class="left-float">
+    <h2> | </h2>
+  </div>
+  <div class="left-float">
+    <h2>  Sessions </h2>
+  </div>
+  <div class="right-float">
+    <button class="ui button"> Preview </button>
+    <button class="ui button"> Publish </button>
+    <button class="ui button"> Copy </button>
+    <button class="ui red button"> Delete </button>
+  </div>
+</div>
+<br>
+<br>
+<br>
+<div>
+  <div class="ui buttons">
+    <button class="ui button">Overview</button>
+    <button class="ui button">Tickets</button>
+    <button class="ui button">Scheduler</button>
+    <button class="ui active button">Sessions</button>
+    <button class="ui button">Speakers</button>
+    <button class="ui button">Export</button>
+  </div>
+  <div class="ui divider"></div>
+  <br>
```

Comment:
Until here. All this is handled by Issue #77. Please see other issue too to prevent such things.